libs/os/src/main/kotlin/batect/os/ConsoleInfo.kt

```diff
@@ -1,7 +1,7 @@
 /*
     Copyright 2017-2021 Charles Korn.
 
-    Licensed under the Apache License, Version 2.0 (the "License");
+     Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Spotless will complain about this change - you'll need to remove this extra space.

Reply:
Done!